framework/sac_rlhf.py

```diff
@@ -349,7 +349,7 @@ def get_action(self, x):
             intrinsic_reward = knn_estimator.compute_intrinsic_rewards(next_obs)
             knn_estimator.update_states(next_obs)
 
-            rb_exp.add(obs, real_next_obs, actions, intrinsic_reward, terminations, infos)
+            rb_exp.add(obs, real_next_obs, actions, intrinsic_reward, ground_truth_reward, terminations, infos)
```

Comment:
Didn't had a deeper look into the unsuperviced pretraining part yet, @Wizzardexe can you have a look if this is right? Shouldn't have any impact as far as I see.

Reply:
I would remove the extra replay buffer and just use rb.add here.

Reply:
done :+1: